yolort/runtime/y_tensorrt.py

```diff
@@ -228,7 +228,7 @@ def default_loader(self, img_path: str) -> Tensor:
         Returns:
             Tensor, processed tensor for prediction.
         """
-        return read_image(img_path) / 255.0
+        return (read_image(img_path) / 255.0).to(self._device)
```

Comment:
Hi @lauhsu , Could you test the following?
```suggestion
        return read_image(img_path).to(self._device) / 255.0
```